server/v1/controllers/auth.controller.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable arrow-parens */
+/* eslint-disable consistent-return */
+/* eslint-disable prefer-const */
+/* eslint-disable comma-dangle */
+import users from '../models/user.db';
+import user from '../models/user.model';
+import validate from '../helpers/user.validation';
+import auth from '../helpers/authenticate';
+
+class userController {
+  static signUp(req, res) {
+    let { error } = validate(user(req));
+
+    if (error) {
+      return res.status(400).json({
+        status: 'error',
+        error: error.details[0].message.replace(/"/g, ''),
+      });
+    }
+
+    const exist = users.find((usr) => usr.email === req.body.email);
+    if (exist) {
+      res.status(409).json({
+        status: 'error',
+        error: 'This Email already exists',
+      });
+    } else {
+      users.push(user(req));
+      res.status(201).json({
+        status: 'success',
```

Comment:
status refers to the status code, you can call this 'message' and please make it descriptive.

Reply:
Ok!